lib/tiler.ts

```diff
@@ -42,9 +42,14 @@ export async function TileServer<T>({
 
       debug && console.time('query' + id);
       const cacheKey = cache.getCacheKey(table, z, x, y, filtersQuery);
-      const value = await cache.getCacheValue(cacheKey);
-      if (value) {
-        return value;
+      try {
```

Comment:
should we do this inside cache.ts maybe?

Reply:
There we don't have the debug flag.